Makefile

```diff
@@ -2,7 +2,7 @@
 # are usually completed in github actions.
 
 SHELL := /bin/bash
-VERSION := 1.1.3
+VERSION := 1.2
```

Comment:
This says version 1.2, but in the Changelog the change is under Version 1.1.4

Reply:
Edited to 1.2 in Changelog.